packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -1246,7 +1246,7 @@ void main() {
 
     bool currentIsCompleted = controller.value.isCompleted;
 
-    final void Function() isCompletedTest = expectAsync0(() {}, count: 2);
+    final void Function() isCompletedTest = expectAsync0(() {});
```

Comment:
It is unclear to me why this is changing, or how this test is related to the behavior your PR is intended to fix. This PR should be adding one or more new tests that clearly cover the specific behavior you are fixing.

Reply:
@stuartmorgan updated that test and added one new test

Reply:
@stuartmorgan turns out that test was failing as was not in a play state as test implied

Reply:
@stuartmorgan are these changes okay ?